src/System.Security.Cryptography.Cng/tests/CreateTests.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public static class CreateTests
+    {
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public static void CreateEmphemeral1()
```

Comment:
I don’t know what the [Theory] attribute does but given that this is a lightweight interop layer over CNG, I think it’s a second order priority for the CoreFx tests to test all the algorithms (we’d be testing the underlying CNG platform rather than getting code coverage on CoreFx itself.)



From: Jeremy Barton [mailto:notifications@github.com]
Sent: Tuesday, August 4, 2015 4:07 PM
To: dotnet/corefx
Cc: Atsushi Kanamori
Subject: Re: [corefx] Implement System.Security.Cryptography.CngKey (#2615)


In src/System.Security.Cryptography.Cng/tests/CreateTests.cs<https://github.com/dotnet/corefx/pull/2615#discussion_r36253278>:

> @@ -0,0 +1,55 @@

> +// Copyright (c) Microsoft. All rights reserved.

> +// Licensed under the MIT license. See LICENSE file in the project root for full license information.

> +

> +using Test.Cryptography;

> +using Xunit;

> +

> +namespace System.Security.Cryptography.Cng.Tests

> +{

> +    public static class CreateTests

> +    {

> +        [Fact]

> +        [PlatformSpecific(PlatformID.Windows)]

> +        public static void CreateEmphemeral1()

Should these tests be [Theory] over various CngAlgorithms?

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/corefx/pull/2615/files#r36253278>.